controllers/ibu_controller.go

```diff
@@ -242,11 +242,11 @@ func (r *ImageBasedUpgradeReconciler) handleAbortOrFinalize(ctx context.Context,
 		case string(utils.ConditionReasons.Aborting):
 			nextReconcile, err = r.handleAbort(ctx, ibu)
 		case string(utils.ConditionReasons.AbortFailed):
-			nextReconcile, err = r.handleAbortFailure(ctx, ibu)
+			nextReconcile, err = r.handleIdleFailure(ctx, ibu)
 		case string(utils.ConditionReasons.Finalizing):
 			nextReconcile, err = r.handleFinalize(ctx, ibu)
```

Comment:
Should we now return immediate requeue in handleFinalize/handleAbort for failed cleanup case? otherwise I think the IBU status will be reset by the next `ResetStatusConditions` func.

Reply:
Done. Also changed to condition to reset status to only when reason is idle